go.mod

```diff
@@ -377,3 +376,7 @@ replace (
 
 // Exclude this version because the version has an invalid checksum.
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+replace github.com/elastic/elastic-agent-client/v7 => github.com/AndersonQ/elastic-agent-client/v7 v7.0.0-20230308191952-d7d45d93d0da
```

Comment:
i dont' really like this

Reply:
i see it's not in yet, doing PR reviews in reverse. my bad

Reply:
No worries.  Indeed, once https://github.com/elastic/elastic-agent-client/pull/56 is merged, we can remove this line.

Reply:
Removed.